core/services/helper/html/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Helper</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+
+    <!-- CSS and JS -->
+    <link rel="stylesheet" href="static/css/metro-all.min.css" />
+    <script src="static/js/axios.min.js"></script>
+    <script src="static/js/metro.min.js"></script>
+    <script src="static/js/vue.js"></script>
+
+    <style>
+      body {
+        min-height: 75rem
+        padding-top: 4.5rem
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="app" class="container pos-fixed pos-top-center">
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Port</th>
+            <th>Service Name</th>
+            <th>Webpage</th>
+            <th>API Documentation</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="service in services">
+            <td>{{service.port}}</td>
+            <td>{{service.title}}</td>
+            <td v-html="createWebpageHyperlink(service.port)"></td>
+            <td
+              v-if="service.documentation"
+              v-html="createWebpageHyperlink(service.port, service.documentation)"
+            ></td>
+            <td v-else>No API documentation</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+
+    <script>
+      /*global Metro, Vue, axios*/
+      window.app = new Vue({
+        el: '#app',
+        mounted: function () {
+          // Start metro
+          Metro.init()
+
+          // Fetch network data
+          this.requestData()
+
+          // Fetch periodic API request
+          this.startPeriodicRequest()
+        },
+        data: function () {
+          return {
+            services: [],
+          }
+        },
+        methods: {
+          createUrlHyperlink: function (url) {
+            return `<a href="${url}">${url}</a>`
+          },
+          createWebpageUrl: function (port, path = '') {
+            return `${window.location.protocol}//${
+              window.location.host.split(':')[0]
+            }:${port}${path}`
+          },
+          createWebpageHyperlink: function (port, path = '') {
```

Comment:
```suggestion
          createWebpageHyperlink(port, path = '') {
```

"function" is unnecessary here

Reply:
Check #223 to avoid such pain in the future.